mqtt/mqtt-bridge/src/persist/waking_state/mod.rs

```diff
@@ -275,6 +277,8 @@ mod tests {
         // insert an element to wake the stream, then wait for the other thread to complete
         let mut state_borrow = state.borrow_mut();
         state_borrow.insert(key1, pub1).unwrap();
+        drop(state_borrow);
```

Comment:
why?

Reply:
I thought it made this more clear that the allow was because of a false positive, but I guess it is ok as it was before.